### ISSUE TITLE
Bring all `ScalarBigXXXStore` methods into consistency with the types defined in the `stores` package

### ISSUE THREAD

**Issue by erights** [Agoric/Agoric#7632]
The MapStore and SetStore `addAll` methods are correctly typed and implemented by `@agoric/store`. But it lacks both explanation and tests.

The virtual and durable MapStores fail to implement `addAll` at all. The virtual and durable SetStores implementations of `addAll` are incomplete: they correctly recognize and process an `Iterable` argument. But they do not recognize a `CopySet` argument.

**Comment by erights** [comment]
Also `snapshot` methods such as https://github.com/Agoric/agoric-sdk/blob/09b993a22a1618fc6c0e3bddb1d09b6fa4e78de5/packages/store/src/stores/scalarMapStore.js#L100-L101 are implemented only by heap stores, but not by virtual or durable ones. These are the dual of the `addAll` methods, so it makes sense that they were omitted together.

**Comment by FUDCo** [comment]
> The virtual and durable SetStores implementations of `addAll` are incomplete: they correctly recognize and process an `Iterable` argument. But they do not recognize a `CopySet` argument.

I don't understand what this means.  Are `CopySet`s not iterable?

**Comment by FUDCo** [comment]
As I was working through how to apply the fixes for this, I discovered another error: it appears we have an internal-only operation on sets (`addToSet`) being added to the public API of all the collection classes.  This seems to be the consequence of some unrelated cleanup.  This was introduced in commit c81d3677d8085eb4debe5baa416816ff94d582cf (I poked around in the git log to see what idiot had done this and of course it was me).  If @erights concurs I'd like to recast this issue as "Bring all `ScalarBigXXXStore` methods into consistency with the types defined in the `stores` package".
